K2Bridge.Tests.UnitTests/JsonConverters/AggregationConverterTests.cs

```diff
@@ -95,23 +101,23 @@ public class AggregationConverterTests
                     { "2", new Aggregation() {
                         PrimaryAggregation = new AvgAggregation {
                             FieldAlias = "_2",
-                            FieldName = "grade",
+                            FieldName = "metric",
                             },
                         SubAggregations = new Dictionary<string, Aggregation>(),
                         }
                     },
                 },
         };
 
-        private static readonly Aggregation ExpectedNoFieldsAvgAggregation = new Aggregation()
+        private static readonly Aggregation ExpectedValidSumAggregation = new Aggregation()
         {
             PrimaryAggregation = null,
             SubAggregations = new Dictionary<string, Aggregation>
                 {
                     { "2", new Aggregation() {
```

Comment:
Why is this replacing the 'avg aggregation' instead of adding to it

Reply:
I documented the point in the PR. I removed one test I considered not really relevant -> JsonDeserializeObject_WithAggregationNoFieldsAvg_DeserializedCorrectly. This test was done only for the average metric, and I am not sure it really covers a real need. But I can add it back if we consider the code coverage is too low.

Reply:
+1 on returning the test

Reply:
Added back.